tests/unit/amo/components/TestSearch.js

```diff
@@ -73,28 +77,39 @@ describe('<Search />', () => {
     expect(sort.prop('pathname')).toEqual(props.pathname);
   });
 
-  it('does not render a SearchSort when there are no filters', () => {
-    const root = render({ hasSearchParams: false, results: [] });
+  it('does not render a SearchSort when there are no results', () => {
+    const root = render({ count: 0 });
```

Comment:
Tweaking this test turned out to be a bit more complicated and is a big change in terms of lines. I think it's worth doing (and did it for this test) but a few are more complicated... I'd rather leave it for another patch if that's okay.